### DISCUSSION TITLE
New HMS beta

### DISCUSSION THREAD

**Discussion by jscherry** [SecKatie/ha-wyzeapi#180] [Q&A] [answered]
Does this also bring the V2 motions and V2 contact sensors into home assistant where they can be used for basic automations or does it only work with the new bridge? Very insterested, just wanted to verify before installing. Thanks

**Comment by SecKatie** [comment, marked as answer]
Currently it just brings the ability to arm, set home, or away. I will be adding sensors next.

**Comment by jscherry** [comment]
Thanks
